build.tmpl.yml

```
@@ -228,66 +225,134 @@ archives:
   - id: windows-cgo
     builds:
       - windows-cgo
-    replacements:
-      darwin: macOS
-      386: 32bit
-      amd64: 64bit
     format_overrides:
       - goos: windows
         format: zip
-    name_template: "{{ .ProjectName }}_{{ .Version }}-{{ .Os }}_sqlite_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+    name_template: >-
+      {{- .ProjectName }}_
+      {{- .Version }}-
+      {{- if .Os eq "darwin" }}macOS
+      {{- else }}{{ .Os }}
+      {{- end }}-sqlite_
```

Looks like I commented on the wrong file:

```suggestion
      {{- end }}_sqlite_
```